src/main/java/mate/jdbc/model/Car.java

```diff
@@ -0,0 +1,83 @@
+package mate.jdbc.model;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Car {
+    private Long id;
+    private String model;
+    private Manufacturer manufacturer;
+    private List<Driver> drivers;
+
+    public Car() {
+    }
+
+    public Car(Long id, String model, Manufacturer manufacturer, List<Driver> drivers) {
```

Comment:
do you use this constructor somewhere?

Reply:
In main and in carParser

Reply:
you don't use this constructor, take a look